src/views/components/Legend/Legend.js

```diff
@@ -31,10 +31,9 @@ Legend.propTypes = {
   title: PropTypes.string.isRequired,
   items: PropTypes.arrayOf(
     PropTypes.shape({
-      name: PropTypes.string.isRequired,
-      color: PropTypes.string.isRequired
```

Comment:
We should keep color here because the code in this component needs the color property.